package.json

```diff
@@ -70,7 +71,10 @@
     "luma.gl": "^7.3.2",
     "mapbox-gl": "^1.13.0",
     "mapbox-gl-compare": "^0.4.0",
+    "material-ui-popup-state": "^4.1.0",
+    "moment": "^2.29.4",
```

Comment:
I can not see why the package `moment` got added to the packages again as it is not used anymore?